src/sql/engine/expr/ob_expr_st_area.cpp

```diff
@@ -101,7 +101,7 @@ int ObExprSTArea::eval_st_area(const ObExpr &expr, ObEvalCtx &ctx, ObDatum &res)
         ObGeoExprUtils::geo_func_error_handle(ret, N_ST_AREA);
       } else if (!std::isfinite(result)) {
         ret = OB_OPERATE_OVERFLOW;
-        LOG_WARN("Result value is out of range in st_area", K(ret));
+        LOG_USER_ERROR(OB_OPERATE_OVERFLOW, "Result", N_ST_AREA);
```

Comment:
advised to reserve LOG_WARN because logs generated by LOG_USER_ERROR do not contain module information like [SQL]

Reply:
@Young-Cody  please resolve these comments, so that we can continue to merge this PR.

Reply:
Done.